### ISSUE TITLE
it is not possible to see the repo of which I am a collaborator

### ISSUE THREAD

**Issue by rgarofalo** [go-gitea/gitea#6370]
- Gitea version (or commit ref): 91775c1
- Git version: 2.20.1
- Operating system: Linux Ubuntu server 18.10 
- Database (use `[x]`):
  - [ ] PostgreSQL
  - [x] MySQL
  - [ ] MSSQL
  - [ ] SQLite
- Can you reproduce the bug at https://try.gitea.io:
  - [ ] Yes (provide example URL)
  - [x] No
  - [ ] Not relevant
- Log gist:

## Description

users collaborating in a private repo inserted in a private organization cannot view the repositories in the explore/repos and home organization

**Comment by mvrhov** [comment]
Yeah, this seems to be an oversight with the organization implementation. Ping @techknowlogick

**Comment by kolaente** [comment]
Users can see the repos on the org home page, but only if they are a member of a team which has access to the org, not if they are only added as a collaborator.
